tooling/sync/Makefile

```diff
@@ -186,3 +186,39 @@ start-ethrex: ## Start ethrex for the network given by NETWORK.
     		--authrpc.jwtsecret $(DATA_PATH)/${NETWORK}_data/jwt.hex \
     		$(BOOTNODES_FLAG) \
     		--evm $(EVM)
+
+SERVER_SYNC_BRANCH ?= snap_sync
+SERVER_SYNC_NETWORK ?= hoodi
+
+ifeq ($(SERVER_SYNC_NETWORK),hoodi)
+CHECKPOINT_URL = https://hoodi-checkpoint-sync.attestant.io/
+else ifeq ($(SERVER_SYNC_NETWORK),sepolia)
+CHECKPOINT_URL = https://checkpoint-sync.sepolia.ethpandaops.io
+else ifeq ($(SERVER_SYNC_NETWORK),mainnet)
+CHECKPOINT_URL = https://mainnet-checkpoint-sync.attestant.io
+else
+$(error Unknown network $(SERVER_SYNC_NETWORK))
+endif
+
+LOGS_FILE ?= output.log
+
+# Use make server-sync SERVER_SYNC_BRANCH=branch_name SERVER_SYNC_NETWORK=network_name LOGS_FILE=logs_file_name  HEALING=1 MEMORY=1 SNAP=1
+# SERVER_SYNC_BRANCH is the branch to checkout before syncing, SERVER_SYNC_NETWORK is the network to sync, LOGS_FILE is the file to output logs to, HEALING enables healing mode, MEMORY uses memory datadir, SNAP enables snap sync mode.
+server-sync:
+	git fetch --all
+
+	git checkout $(SERVER_SYNC_BRANCH)
+
+	git pull
+
+	tmux kill-server || true
```

Comment:
This is a bit aggressive and breaks the workflow for tmux users. Just kill the session that ethrex controls.